tasks/main.yml

```diff
@@ -84,7 +87,24 @@
     body_format: json
     body: "{{ item }}"
     force_basic_auth: true
-  with_items: "{{ grafana_conf_available_datasources|extract_new_datasources(grafana_conf_existing_datasources['content']) }}"
+  with_items: "{{ grafana_conf_enabled_datasources|extract_new_datasources(grafana_conf_existing_datasources['content']) }}"
+  changed_when: True
+
+- name: Ensure a directory exists to clone the dashboard repo into
+  file:
+    path: "{{ grafana_conf_grafana_dashboard_repo.checkout_path }}"
+    state: directory
+    mode: 0755
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_id }}"
+  become: True
+  when: grafana_conf_grafana_dashboard_repo | bool
+
+- name: Clone dashboards
+  git:
+    repo: "{{ grafana_conf_grafana_dashboard_repo.path }}"
+    dest: "{{ grafana_conf_grafana_dashboard_repo.checkout_path }}"
```

Comment:
Should we also support an optional `.version`?

Reply:
Done